.gitlab-ci.yml

```diff
@@ -0,0 +1,55 @@
+stages:
```

Comment:
I'm fine with this, but I thought @PeterMosmans wanted this in a separate repo

Reply:
CI/CD config has been removed.